src/Models/RaspberryPiDevice.ts

```diff
@@ -221,10 +221,15 @@ export class RaspberryPiDevice implements Device {
           RaspberryPiUploadConfig.user, RaspberryPiUploadConfig.password);
       try {
         await ssh.uploadFile(binFilePath, RaspberryPiUploadConfig.projectPath);
-        const enableExecPriorityCommand =
-            `cd ${RaspberryPiUploadConfig.projectPath} && chmod -R 755 .\/`;
+        const binFileName = path.basename(binFilePath);
+        const enableExecPriorityCommand = `cd ${

```

Comment:
kill the running process before copy

Reply:
add `killall` command to try to kill previous process